Automattic/sensei
# Create translations for questions in category

While we translate a lesson and create translations (duplicates) for questions we also need to create translations (duplicates) for lessons in categories we attach to the quiz.

_Warning:_
Unfortunately, questions are not translated automatically. (You can see these strings in the WPML translation editor, but they don't appear in the translated lesson.)
For now, the user has to translate them manually in lessons: open the translated lesson in the WordPress editor and change texts for questions.

## Proposed Changes
* Create translations for questions in category

## Testing Instructions

0. Note that we translate the course on step 9, we don't create other translations earlier.
1. Install, activate and set up WPML, etc.
2. Go to Sensei LMS -> Questions -> Question Categories.
3. Create a new question category.
4. Go to Sensei LMS -> Questions.
5. Create two questions attach them to the category you created on step 3.
6. Create a course with a lesson.
7. Open the lesson in the WP editor.
8. Add a quiz and add a question category with the category you created on step 3.
9. Translate the course.
10. Make sure you have translations of questions from the category for all languages.
11. Check the quiz on the frontend.


## Pre-Merge Checklist
